packaging/docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM centos:7
+
+RUN yum install -y deltarpm
+RUN yum install -y epel-release
+RUN yum install -y initscripts sudo openssh openssh-clients openssh-server \
+	systemd-sysv openssl-1.0.2k libselinux-utils logrotate java python-setuptools \
+	python-backports python-backports-ssl_match_hostname which
+RUN ssh-keygen -A
+
+ENV container=docker
+
+RUN (cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i == systemd-tmpfiles-setup.service ] || rm -f $i; done); \
+rm -f /lib/systemd/system/multi-user.target.wants/*;\
+rm -f /etc/systemd/system/*.wants/*;\
+rm -f /lib/systemd/system/local-fs.target.wants/*; \
+rm -f /lib/systemd/system/sockets.target.wants/*udev*; \
+rm -f /lib/systemd/system/sockets.target.wants/*initctl*; \
+rm -f /lib/systemd/system/basic.target.wants/*;\
+rm -f /lib/systemd/system/anaconda.target.wants/*;
+
+EXPOSE 22 80 443 5671 5672 15672 9200 5432 8086 999
+
```

Comment:
might also need the SIGTERM+3 systemd hack to be able to stop the container gracefully: ` STOPSIGNAL SIGRTMIN+3`
Like eg. here: https://hub.docker.com/r/solita/centos-systemd/~/dockerfile/